apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -972,6 +972,32 @@ const Ticket = new GQLListSchema('Ticket', {
         delete: false,
         auth: true,
     },
+    kmigratorOptions: {
+        indexes: [
+            {
+                type: 'BTreeIndex',
+                fields: ['unitName'],
+                name: 'ticket_unitName',
+            },
+            {
+                type: 'BTreeIndex',
+                fields: ['assignee', 'executor', 'organization'],
```

Comment:
Probably, if we are going to use such query certain indexes - let's add a comment to the source code, that use such queries: just to notify developers, they must update an index, in case if they will change query in source code

Reply:
What about order? Should it start from organization? Or does we fix specific case?

Reply:
yes, Pasha added comment in pages/ticket/index.js